_source/reference/attributes.md

```diff
@@ -29,6 +29,7 @@ The following attributes are automatically added by Turbo and are useful to dete
 
 * `disabled` is added to the form submitter while the form request is in progress, to prevent repeat submissions.
 * `data-turbo-preview` is added to the `html` element when displaying a [preview](/handbook/building#detecting-when-a-preview-is-visible) during a Visit.
+* `data-turbo-visit-direction` is added to the `html` element during a view transition, with a value of `forward` or `back` or `none`, to indicate its direction.
```

Comment:
Although this will probably we used for customising view transitions, technically it's added for the duration of the visit, not just view transitions. (I don't think it really matters in the handbook, but perhaps worth clarifying here.)

```suggestion
* `data-turbo-visit-direction` is added to the `html` element during a visit, with a value of `forward` or `back` or `none`, to indicate its direction.
```